blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -74,6 +74,11 @@ Changes
 Fixes
 =====
 
+- Updated the admin-ui to version 1.15.1 to fix the partitions in the table view
```

Comment:
Now it contains even more implementation details...

```suggestion
- Fixed an issue in the admin ui that prevented partitions from showing up in the table detail view.
```

Or something like that.